src/sentry/api/endpoints/organization_users.py

```diff
@@ -2,14 +2,35 @@
 
 from rest_framework.response import Response
 
+from sentry.api.base import DocSection, EnvironmentMixin
 from sentry.api.bases.organization import OrganizationEndpoint
 from sentry.api.serializers import serialize
 from sentry.api.serializers.models import OrganizationMemberWithProjectsSerializer
 from sentry.models import OrganizationMember
+from sentry.utils.apidocs import scenario, attach_scenarios
 
 
-class OrganizationUsersEndpoint(OrganizationEndpoint):
+@scenario("ListOrganizationUsers")
+def list_organization_users_scenario(runner):
+    runner.request(method="GET", path="/organizations/%s/users/" % runner.org.slug)
+
+
+class OrganizationUsersEndpoint(OrganizationEndpoint, EnvironmentMixin):
+    doc_section = DocSection.ORGANIZATIONS
+
+    @attach_scenarios([list_organization_users_scenario])
     def get(self, request, organization):
+        """
+        List an Organization's Users
+        ````````````````````````````
+
+        Return a list of users that belong to a given organization.
+
+        :qparam string project: restrict results to users who have access to a given project ID
+        :pparam string organization_slug: the slug of the organization for which the users
+                                          should be listed.
+        :auth: required
+        """
```

Comment:
This endpoint was added manually: https://github.com/getsentry/sentry-docs/pull/1336. So, added it here.